bin/git-missing

```diff
@@ -12,16 +12,28 @@ fi
 
 declare -a git_log_args=()
 declare -a branches=()
+declare pathspec=""
+declare parse_path=false
+
 for arg in "$@" ; do
 
+    if [[ $parse_path == true ]]; then
+        pathspec="$*"
+        break
+    fi
+
     case "$arg" in
+        --)
+            parse_path=true
+            ;;
         --*)
             git_log_args+=( "$arg" )
             ;;
         *)
             branches+=( "$arg" )
             ;;
     esac
+    shift
```

Comment:
Was this unintentional? I don't think this changes the behavior of the program at all and it could be removed

Reply:
Yes, it's a leftover from my tinkering, sorry about that. Removed. (b8488cd)